app/src/main/java/com/sunasterisk/musixmatch/ui/search/MySuggestionProvider.java

```diff
@@ -0,0 +1,17 @@
+package com.sunasterisk.musixmatch.ui.search;
+
+import android.content.SearchRecentSuggestionsProvider;
+
+/**
+ * Created by superme198 on 20,March,2019
+ * in Music_Match_1.
+ */
+public class MySuggestionProvider extends SearchRecentSuggestionsProvider {
```

Comment:
```suggestion
public class SuggestionProvider extends SearchRecentSuggestionsProvider {
```
ko nên đặt các class hay biến có tên là my .... gì đó
nó k mang nhiều giá trị